blogposts/2021/better-onboarding-prevent-codebase-overwhelm.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "Better onboarding: How to prevent codebase overwhelm"
+description: "A data-driven look at how the rise in code complexity affects new developers joining a team, and how engineering managers can support new hires with better onboarding."
+author: Patricia No and Rebecca Dodd
+authorUrl: 
+publishDate: 2021-04-29T18:00+02:00
+tags: [blog, remote, design]
```

Comment:
@content-hacker Do we have a list of supported tags anywhere?

Reply:
Not that I know of. I'm not sure they even show up anywhere on the blog